man/mptcpd.8.in

```diff
@@ -1,11 +1,11 @@
 .\" SPDX-License-Identifier: BSD-3-Clause
 .\"
-.\" Copyright (c) 2017-2019, Intel Corporation
+.\" Copyright (c) 2017-2020, Intel Corporation
 
 .\" Process this file with
 .\" groff -man -Tascii mptcpd.8
 .\"
-.TH MPTCPD 8 "2018-09-13" "Multipath TCP Daemon" "System Management Commands"
+.TH MPTCPD 8 "2020-07-10" "Multipath TCP Daemon" "System Management Commands"
```

Comment:
Should this date be current?

Reply:
The date corresponds to the date that I made the actual change to the man page.  I'm okay with leaving it as is.